doc/source/conf.py

```diff
@@ -193,7 +194,7 @@ def reset_servers(gallery_conf, fname, when):
 # -- Options for HTML output -------------------------------------------------
 html_short_title = html_title = "PyDPF-Core"
 html_theme = "ansys_sphinx_theme"
-html_logo = pyansys_logo_black
+html_logo = pyansys_logo_light_mode
```

Comment:
```suggestion
```
yes you need to remove this line, and then add "logo":"pyansys", in html theme options. That should work

Reply:
I just test this variation and it still gives me the same error

Reply:
let me try it locally.

Reply:
yes, you are correct, and i think it is because of some mismatch in configuration. i can fix with with another workaround and push that changes if its ok for you